python/taichi/lang/transformer.py

```diff
@@ -332,7 +333,9 @@ def visit_For(self, node):
         __I = __ndrange_I
             '''.format(node.target.id, dim)
             t = ast.parse(template).body[0]
-            t.body[0].value = node.iter if is_ndrange_for == 1 else node.iter.args[0]
+            t.body[
+                0].value = node.iter if is_ndrange_for == 1 else node.iter.args[
+                    0]
```

Comment:
It looks bad here...

Reply:
Yeah, we should probably break it down into shorter lines :-)